recipe/activate.sh

```diff
@@ -14,6 +14,7 @@ then
     export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
     export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
     export LDFLAGS="${LDFLAGS} -lc++"
+    export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib" 
```

Comment:
Could we please add this for Linux too? Would change to `-Wl,-rpath=$PREFIX/lib`.